### PR TITLE
Make DORMANT_MSG Dynamic

### PR DESCRIPTION
**Pull request by TizzySaurus**
Solves #1736

`DORMANT_MSG` now has the category names dynamically set.

### CODE REVIEW

**Comment by NIRDERIi** [comment]
Maybe try to somehow get the GuildChannel object and check its category? Not sure why it wouldn't be in cache tho

File: bot/exts/help_channels/_cog.py [hunk 1]
```diff
@@ -387,7 +387,14 @@ async def move_to_dormant(self, channel: discord.TextChannel) -> None:
         )
 
         log.trace(f"Sending dormant message for #{channel} ({channel.id}).")
-        embed = discord.Embed(description=_message.DORMANT_MSG)
+        available_category = self.bot.get_channel(constants.Categories.help_available)
```
Comment (Bluenix2) [inline bot/exts/help_channels/_cog.py:390]: You should probably check for None and in that case add a log, and default to the old Dormant channels text.

Comment (TizzySaurus) [reply bot/exts/help_channels/_cog.py]: I've instead used `utils.channels.try_get_channel`, as suggested by Chrisjl on discord, which will always get the channel

**Review by NIRDERIi — APPROVED** [review]
The updated code should be working fine by the looks of it!

File: bot/exts/help_channels/_cog.py [hunk 2]
```diff
@@ -387,7 +387,14 @@ async def move_to_dormant(self, channel: discord.TextChannel) -> None:
         )
 
         log.trace(f"Sending dormant message for #{channel} ({channel.id}).")
-        embed = discord.Embed(description=_message.DORMANT_MSG)
+        available_category = await channel_utils.try_get_channel(constants.Categories.help_available)
+        embed = discord.Embed(
+            description=_message.DORMANT_MSG.format(
+                dormant=channel.category.name,
```
Comment (ChrisLovering) [inline bot/exts/help_channels/_cog.py:393]: Testing this locally, it seems the channel object here isn't actually updated with the new category when it's moved on the lines above, so the embed reads as below. As you can see, it still references the in use category.

image

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

**Review by ChrisLovering — APPROVED** [review]
Re-tested, all works 👌

File: bot/exts/help_channels/_cog.py (hunk 2 above)
Comment (TizzySaurus) [reply bot/exts/help_channels/_cog.py]: Fixed in [this](https://github.com/python-discord/bot/pull/1738/commits/99bfb15037ce7a6f88a403f8c1d2ab36d852874a) commit.

**Review by brad90four — APPROVED** [review]
For what it's worth, looks good to me

**Pull request merged**